src/main/kotlin/no/nav/personbruker/dittnav/eventaggregator/metrics/db/DBMetricsProbe.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.personbruker.dittnav.eventaggregator.metrics.db
+
+import no.nav.personbruker.dittnav.eventaggregator.config.EventType
+import no.nav.personbruker.dittnav.eventaggregator.metrics.MetricsReporter
+import no.nav.personbruker.dittnav.eventaggregator.metrics.PrometheusMetricsCollector
+import no.nav.personbruker.dittnav.eventaggregator.metrics.influx.EVENTS_CACHED
+
+class DBMetricsProbe(private val metricsReporter: MetricsReporter) {
+
+    suspend fun numberOfCachedEventsOfType(numberOfEvents: Int, eventType: EventType) {
+        metricsReporter.registerDataPoint(
+                EVENTS_CACHED,
+                listOf("numberOfEvents" to numberOfEvents).toMap(),
+                listOf("eventType" to eventType.eventType).toMap()
```

Comment:
Hadde det vært ønskelig å skille på systembrukere også? Når det gjelder den periodiske feilen vi har hatt mot basen gir det ikke så mye, ettersom det skjer uavhengig av produsent. Men det kunne vært til hjelp til f. eks. å identifisere produsenter som generer done events for eventer de aldri sendte i utgangspuntet.

Reply:
Ja, det er et bra poeng. Legger til det.